.env.example

```
@@ -1,5 +1,5 @@
 # The version of singularity you will use to make deals. Generally, this should not change
-SINGULARITY_REF=:v0.4.1
+SINGULARITY_REF=:v0.5.0
```

```suggestion
SINGULARITY_REF=:v0.5.1
```
Let's use v0.5.1, because v0.5.0 does not pass integration tests.